charts/linkerd-crds/templates/gateway.networking.k8s.io_grpcroutes.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     gateway.networking.k8s.io/bundle-version: v1.1.1
     gateway.networking.k8s.io/channel: experimental
     {{ include "partials.annotations.created-by" . }}
+    helm.sh/resource-policy: keep
```

Comment:
tioli: may be wise to split this change into a separate PR: the values change is purely additive. This changes behavior somewhat..